specification/v3.2/OSDM-online-api-v3.2.0_draft.yml

```diff
@@ -10257,6 +10330,24 @@ components:
           items:
             $ref: '#/components/schemas/VAT'
 
+    PricingReference:
```

Comment:
I would use `AvailabilityReference` or `AvailabilityCriteria`. _Criteria_ is a common term for offer request, and this multiple place maps are provided not only because of pricing, but also because of linking different reservations and places available for them to occupy.